accounting/src/main/java/io/skai/accounting/controller/RepairController.java

```diff
@@ -0,0 +1,18 @@
+package io.skai.accounting.controller;
+
+import io.skai.accounting.dto.repair.RepairRequestDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/repair")
+@Slf4j
+public class RepairController {
+    @PostMapping
```

Comment:
??

Reply:
it will be done in next iteration. I started to do it and then switched to WAREHOUSE